launch_ros/launch_ros/actions/composable_node_container.py

```diff
@@ -18,14 +18,19 @@
 from typing import Optional
```

Comment:
[Question]
@ivanpauno @hidmic By the way, what is the correct way to format files?

I used this `.isort.cfg` and VSCode settings, but there were some diffs.

```ini
[settings]
force_sort_within_sections=true
force_single_line=true
reverse_relative=true
order_by_type=false
known_third_party=launch
```

```json
{
  "python.formatting.provider": "autopep8",
  "python.formatting.autopep8Args": ["--max-line-length=99"],

  "python.linting.flake8Enabled": true,
  "python.linting.flake8Args": [
    "--extend-ignore=C816,D100,D101,D102,D103,D104,D105,D106,D107,D203,D212,D404,I202",
    "--import-order-style=google",
    "--max-line-length=99",
    "--show-source",
    "--statistics"
  ],
  "python.sortImports.args": ["--settings-path", "${workspaceFolder}/.isort.cfg"],
  "editor.codeActionsOnSave": {
    "source.organizeImports": true
  }
}
```

![image](https://user-images.githubusercontent.com/31987104/117403337-f8b13f00-af42-11eb-8e6b-6160ebb3ed83.png)
![image](https://user-images.githubusercontent.com/31987104/117403351-fe0e8980-af42-11eb-8bd2-6b8ed4f576c3.png)

Reply:
And here? I don't know why there is a newline after some `import` and no newline after other `import`.
![image](https://user-images.githubusercontent.com/31987104/117403496-39a95380-af43-11eb-9be7-a828cd8e6b02.png)
![image](https://user-images.githubusercontent.com/31987104/117403507-40d06180-af43-11eb-8837-4d7a33a0fa85.png)

Reply:
I think we're using `flake8`, with the default configuration.
We also use all the flake8 plugins listed [here](https://docs.ros.org/en/rolling/Installation/Ubuntu-Development-Setup.html#install-development-tools-and-ros-tools) (the `flake8-*` packages).